test/integration/targets/netconf_config/tests/iosxr/basic.yaml

```diff
@@ -29,6 +29,6 @@
 
 - assert:
     that:
-      - "'failed' not in result"
+      - "result.failed == false"
```

Comment:
`- result is not failed`

Reply:
I kept it consistent to https://github.com/ansible-collections/ansible.netcommon/blob/main/tests/integration/targets/netconf_config/tests/iosxr/basic.yaml#L43
tests/integration/targets/netconf_config/tests/iosxr/basic.yaml:43

Reply:
I would change the test there too, not all versions of ansible guarantee that 'failed' is in the results, that is why using the test is safer as it combines the test you are removing and the one you are adding into one.